features/sign_up_page.feature

```diff
@@ -9,24 +9,24 @@ Scenario: Submit sign up form [Happy path]
     And I fill in "Email" with "camron@heroku.com"
     And I fill in "Password" with "camron1234"
     And I fill in "Password confirmation" with "camron1234"
-    When I click "Create" to submit form
+    When I click "Create" 
     And I should see "Welcome! You have signed up successfully."
 
 Scenario: Submit sign up form without name [Sad path]
     When I visit the "sign up" page
     Then I fill in "Name" with " "    
-    When I click "Create" to submit form
+    When I click "Create" 
```

Comment:
not a blocker for this pr but when you write the scenario try to follow this pattern
```
- (Given) some context
- (When) some action is carried out
- (Then) a particular set of observable consequences should obtain
```

Reply:
Noted. Thanks.